.github/workflows/verifySiteData.js

```diff
@@ -0,0 +1,97 @@
+/** Script to verify code links in ./.problemSiteData.json */
+
+const fs = require('fs');
+const https = require('/opt/homebrew/lib/node_modules/sync-request');
+
+const PROBLEMS_SITE_DATA = JSON.parse(fs.readFileSync('./.problemSiteData.json', 'utf8'));
+
+const languageMap = {
+    c: {
+        name: 'C',
+        directory: 'c',
+        extension: 'c'
+    },
+    cpp: {
+        name: 'C++',
+        directory: 'cpp',
+        extension: 'cpp'
+    },
+    csharp: {
+        name: 'C#',
+        directory: 'csharp',
+        extension: 'cs'
+    },
+    java: {
+        name: 'Java',
+        directory: 'java',
+        extension: 'java'
+    },
+    python: {   
+        name: 'Python',
+        directory: 'python',
+        extension: 'py'
+    },
+    javascript: {
+        name: 'JavaScript',
+        directory: 'javascript',
+        extension: 'js'
+    },
+    typescript: {
+        name: 'TypeScript',
+        directory: 'typescript',
+        extension: 'ts'
+    },
+    go: {
+        name: 'Go',
+        directory: 'go',
+        extension: 'go'
+    },
+    ruby: {
+        name: 'Ruby',
+        directory: 'ruby',
+        extension: 'rb'
+    },
+    swift: {
+        name: 'Swift',
+        directory: 'swift',
+        extension: 'swift'
+    },
+    kotlin: {
+        name: 'Kotlin',
+        directory: 'kotlin',
+        extension: 'kt'
+    },
+    rust: {
+        name: 'Rust',
+        directory: 'rust',
+        extension: 'rs'
+    },
+    scala: {
+        name: 'Scala',
+        directory: 'scala',
+        extension: 'scala'
+    },
+};
+
+const GITHUB_BASE_URL = 'https://github.com/neetcode-gh/leetcode/blob/nc-refactor';
```

Comment:
I guess this can just be the main branch (https://github.com/neetcode-gh/leetcode/blob/main), unless you intentionally left this